### ISSUE TITLE
Cannot get styling to work inside markdown editor

### ISSUE THREAD

**Issue by MenooMeow** [CalebJohn/joplin-rich-markdown#48]
I'm pretty sure this is my mistake but I don't know what I'm doing.  I'm trying to style various markdown editor elements so that color coding makes more sense while editing.

I've tried all the following in userchrome.css to make H1 light blue, but nothing works.  Any suggestions?

```
.CodeMirror.CodeMirror span.cm-header-1 {
    color: #0095ff!important;
}
.cm-header-1, .cm-header-2, .cm-header-3, .cm-header-4, .cm-header-5, .cm-header-6 {
    line-height: 1.5em;
    color: #0095ff!important;
}

span.cm-header.cm-header-1 {
    color: #0095ff!important;
}
```

**Comment by CalebJohn** [comment]
Your first example works for me
```
.CodeMirror.CodeMirror span.cm-header-1 {
    color: #0095ff!important;
}
```

Are you restarting Joplin after making changes to userchrome.css? That's the most common cause of styles not applying. Make sure to use File->Quit, you might have the close to tray icon enabled which means pressing the X button doesn't actually close Joplin.

**Comment by MenooMeow** [comment]
Yes I am.  For some reason, I can add a CSS definition in the browser and it works.  But if I copy/paste into the file, then restart, it doesn't work.  This is not true of every single style, just a few of them.  I'm still doing more testing.  I found that disabling the MacOS theme/plugin, the blank startup failure issue I've been having with the new version of Joplin seems to have gone away.
